imap_processing/cdf/utils.py

```diff
@@ -163,4 +167,5 @@ def write_cdf(dataset: xr.Dataset):
         terminate_on_warning=True,
     )  # Terminate if not ISTP compliant
 
-    return file_path
+    return file_path  # type: ignore[no-any-return]
+    # TODO Change,  Returning Any from function declared to return "list[Path]"
```

Comment:
```suggestion
    return file_path
```

When I tried this on my end, it didn't throw an error, so I think this `ignore` can just be removed?

Reply:
It still throws an error for me, maybe because you disallowed the `no-any-return` error from the comment about the `dict[Any]` declaration.

Reply:
I still get an error on my end...